gettsim/functions_loader.py

```diff
@@ -9,11 +9,11 @@
 def load_user_and_internal_functions(functions):
```

Comment:
```suggestion
def load_user_and_internal_functions(user_functions_raw):
```
Why drop the specific name?